services/venice-router/src/main/java/com/linkedin/venice/router/VeniceRouterConfig.java

```diff
@@ -393,6 +400,12 @@ private void checkProperties(VeniceProperties props) {
     perStoreRouterQuotaBuffer = props.getDouble(ROUTER_PER_STORE_ROUTER_QUOTA_BUFFER, 1.5);
     httpClientOpensslEnabled = props.getBoolean(ROUTER_HTTP_CLIENT_OPENSSL_ENABLED, true);
     identityParserClassName = props.getString(IDENTITY_PARSER_CLASS, DefaultIdentityParser.class.getName());
+    singleGetLongTailRetryBudgetPercentDecimal =
+        props.getDouble(ROUTER_SINGLE_GET_LONG_TAIL_RETRY_BUDGET_PERCENT_DECIMAL, 0.0);
```

Comment:
If the default value is `0`, it means this feature is disabled or all the retry will be rejected?

Reply:
Default value of 0 will disable the RetryManger
```
    if (enforcementWindowInMs <= 0 || retryBudgetInPercentDecimal <= 0) {
      retryBudgetEnabled.set(false);
    }
```